### PR TITLE
remove some redundant log macro

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
Signed-off-by: Lloyd-Pottiger <yan1579196623@gmail.com>

### What problem does this PR solve?

Issue Number: ref #5531 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by hongyunyan — APPROVED** [review]
LGTM

File: dbms/src/Storages/DeltaMerge/ReadThread/CPU.cpp [hunk 1]
```diff
@@ -102,18 +102,18 @@ std::vector<std::vector<int>> getNumaNodes(Poco::Logger * log)
...
     }
     catch (std::exception & e)
     {
-        LOG_FMT_WARNING(log, "{}", e.what());
+        LOG_WARNING(log, "{}", e.what());
     }
     catch (...)
     {
-        LOG_FMT_WARNING(log, "Unknow Error");
+        LOG_WARNING(log, "Unknow Error");
```
Comment (ywqzzy) [inline dbms/src/Storages/DeltaMerge/ReadThread/CPU.cpp:113]:
```suggestion
        LOG_WARNING(log, "Unknown Error");
```

Comment (Lloyd-Pottiger) [reply dbms/src/Storages/DeltaMerge/ReadThread/CPU.cpp]: fixed!

**Review by ywqzzy — APPROVED** [review]

**Pull request merged**